src/dsp_tools/commands/create/create_on_server/complete_ontologies.py

```diff
@@ -0,0 +1,116 @@
+    onto_client = OntologyCreateClientLive(auth.server, auth)
+    overall_success = True
+
+    logger.info("Processing Ontology Section")
+    print(BOLD + "Processing Ontology Section:" + RESET_TO_DEFAULT)
+    project_iri_lookup, ontology_problems = _create_ontologies_on_server(
+        ontologies=parsed_ontologies,
+        project_iri=project_iri,
+        shortcode=shortcode,
+        client=onto_client,
+    )
+    if ontology_problems:
+        overall_success = False
+        print_problem_collection(ontology_problems)
+    if not project_iri_lookup.onto_iris:
+        msg = "No ontologies could be created on the server."
+        logger.error(msg)
+        print(BOLD_RED + "    " + msg + RESET_TO_DEFAULT)
+        return False
+
+    all_classes = []
+    for onto in parsed_ontologies:
+        all_classes.extend(onto.classes)
+    if all_classes:
+        print(BOLD + "Processing Classes:" + RESET_TO_DEFAULT)
+        success_collection, cls_problems = create_all_classes(
+            classes=all_classes,
+            project_iri_lookup=project_iri_lookup,
+            created_iris=success_collection,
+            client=onto_client,
+        )
+        if cls_problems:
+            overall_success = False
+            print_problem_collection(cls_problems)
+    else:
+        logger.info("No classes found in the ontology.")
+
+    all_props = []
+    for onto in parsed_ontologies:
+        all_props.extend(onto.properties)
+    if all_props:
+        print(BOLD + "Processing Properties:" + RESET_TO_DEFAULT)
+        success_collection, property_problems = create_all_properties(
+            properties=all_props,
+            project_iri_lookup=project_iri_lookup,
+            created_iris=success_collection,
+            list_lookup=list_name_2_iri,
+            client=onto_client,
+        )
+        if property_problems:
+            overall_success = False
+            print_problem_collection(property_problems)
+    else:
+        logger.info("No properties found in the ontology.")
+
+    print(BOLD + "Processing Cardinalities:" + RESET_TO_DEFAULT)
+    problems = add_all_cardinalities(
+        ontologies=parsed_ontologies,
+        project_iri_lookup=project_iri_lookup,
+        created_iris=success_collection,
+        onto_client=onto_client,
+    )
+    if problems:
+        overall_success = False
+        print_problem_collection(problems)
+
+    return overall_success
+
+
+def _create_ontologies_on_server(
+    ontologies: list[ParsedOntology], project_iri: str, shortcode: str, client: OntologyCreateClient
+) -> tuple[ProjectIriLookup, CollectedProblems | None]:
+    project_iri_lookup = get_project_iri_lookup(client.server, shortcode, project_iri)
+    new_ontologies = []
+    for parsed_onto in ontologies:
+        if not project_iri_lookup.does_onto_exist(parsed_onto.name):
+            new_ontologies.append(parsed_onto)
+    if not new_ontologies:
+        if project_iri_lookup.onto_iris:
```

Comment:
why? Even if there are no ontos on the server, I expect to get a warning. For me, this would be more logical:


```suggestion
        if len(ontologies) > 0:
```